modules/admin/ref-rbac-rest-api-endpoints.adoc

```diff
@@ -762,6 +834,18 @@ Creates a conditional policy for the specified role.
 |Required
 |===
 
+|`name`
+|Name of the role
+|String
+|Required
+|===
+
+|`metadata.description`
+|The description of the role
+|String
+|Optional
+|===
+
```

Comment:
Could you please check the formatting of the table? It does not look correctly formatted.